oregon-a/basket-dev/worker-deploy.yaml

```diff
@@ -48,366 +37,226 @@ spec:
-            secretKeyRef:
-              key: exacttarget-interests
-              name: basket-dev-v107-env
+          value: GET_INVOLVED
         - name: EXACTTARGET_OPTIN_STAGE
-          valueFrom:
-            secretKeyRef:
-              key: exacttarget-optin-stage
-              name: basket-dev-v107-env
+          value: Double_Opt_In
         - name: EXACTTARGET_PASS
           valueFrom:
             secretKeyRef:
               key: exacttarget-pass
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: EXACTTARGET_USER
           valueFrom:
             secretKeyRef:
               key: exacttarget-user
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: EXACTTARGET_USE_SANDBOX
-          valueFrom:
-            secretKeyRef:
-              key: exacttarget-use-sandbox
-              name: basket-dev-v107-env
+          value: "True"
         - name: EXTRA_SUPPORTED_LANGS
-          valueFrom:
-            secretKeyRef:
-              key: extra-supported-langs
-              name: basket-dev-v107-env
+          value: zh-tw
         - name: FXA_EVENTS_ACCESS_KEY_ID
           valueFrom:
             secretKeyRef:
               key: fxa-events-access-key-id
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: FXA_EVENTS_QUEUE_ENABLE
-          valueFrom:
-            secretKeyRef:
-              key: fxa-events-queue-enable
-              name: basket-dev-v107-env
+          value: "True"
         - name: FXA_EVENTS_QUEUE_REGION
-          valueFrom:
-            secretKeyRef:
-              key: fxa-events-queue-region
-              name: basket-dev-v107-env
+          value: us-east-1
         - name: FXA_EVENTS_QUEUE_URL
           valueFrom:
             secretKeyRef:
               key: fxa-events-queue-url
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: FXA_EVENTS_SECRET_ACCESS_KEY
           valueFrom:
             secretKeyRef:
               key: fxa-events-secret-access-key
-              name: basket-dev-v107-env
+              name: basket-dev-secrets
         - name: HTTPS
+          value: "on"
+        - name: K8S_NAMESPACE
           valueFrom:
-            secretKeyRef:
-              key: https
-              name: basket-dev-v107-env
-        - name: NEWRELIC_INI_FILE
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: K8S_POD_NAME
           valueFrom:
-            secretKeyRef:
-              key: newrelic-ini-file
-              name: basket-dev-v107-env
+            fieldRef:
+              fieldPath: metadata.name
+        - name: NEWRELIC_INI_FILE
+          value: newrelic.ini
         - name: NEW_RELIC_APP_NAME
-          valueFrom:
-            secretKeyRef:
-              key: new-relic-app-name
-              name: basket-dev-v107-env
+          value: basket-dev-oregon-b
```

Comment:
```suggestion
          value: basket-dev-oregon-a;basket-dev
```

Reply:
Assuming this change should also be made to `web-deploy.yaml`?